docs/src/views/Accessibility.tsx

```diff
@@ -152,8 +152,8 @@ export default () => (
     <CodeSnippet
       value={`
         <>
-          <Button icon='envelope' />
-          <Button icon='envelope' aria-label='Send message' />
+          <Button icon='email' />
```

Comment:
As @levithomason mentioned - this is general doc, I would stick with FA here.

Reply:
Are you sure? If user copies these code examples into the `codesandbox` template, the icons won't render.

Reply:
I am not sure, but let's go with what you have now.